CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.12
+
+<!-- generated comparing v5.15.11..master -->
+
+_Mar 4, 2024_
+
+A big thanks to the 16 contributors who made this release possible.
```

Comment:
```suggestion
A big thanks to the 16 contributors who made this release possible. Here is the highlight ✨:
```

Reply:
I didn't keep this because there's only one highlight, and to make the actual highlight more visible ~